### PR TITLE
Tier feature choice design

### PR DESCRIPTION
**Pull request by davidrapson**
This PR adds new templates and styling for feature choice and introduces a *What's Included* panel for all tiers.

**Partner**
partner

**Patron**
patron

**Supporter**
supporter

**Friend**
friend

**Friend to paid**
friend-to-paid

@tudorraul @afiore

### CODE REVIEW

File: frontend/app/model/FeatureChoice.scala [hunk 1]
```diff
@@ -29,11 +28,20 @@ object FeatureChoice {
 
 case object Books extends FeatureChoice {
   override val zuoraCode = "Books"
-  override val label = "4 free books"
+  override val label = "4 books"
+  override val description = """
+    |Get six Guardian Live tickets to use throughout the year.
```
Comment (afiore) [inline frontend/app/model/FeatureChoice.scala]: Feature descriptions are inverted!

Comment (davidrapson) [inline frontend/app/model/FeatureChoice.scala]: Just testing…

File: frontend/app/model/FeatureChoice.scala [hunk 2]
```diff
@@ -29,11 +28,20 @@ object FeatureChoice {
...
+    |Use one ticket per event at the event of your choosing.
+  """.stripMargin
 }
+
 case object FreeEventTickets extends FeatureChoice {
   val allowance = zuoraFreeEventTicketsAllowance
   override val zuoraCode = "Events"
-  override val label = s"$allowance free events"
+  override val label = s"$allowance tickets"
+  override val description = """
```
Comment (afiore) [inline frontend/app/model/FeatureChoice.scala]: Also, is there a reason why we use a word in one description and a number in the other? If we decide to go for numbers, can we please render the `$allowance` value into the description? I have made this value configurable to facilitate QA work

File: frontend/app/views/fragments/form/featureChoiceFieldset.scala.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                      @choice.description
+                    </div>
+                </div>
+            }
+        </div>
+    </fieldset>
+}
+
+@if(isTestUser(request.user) || Config.stageDev) {
+    @if(Seq(Patron, Partner).contains(tier)) {
```
Comment (afiore) [inline frontend/app/views/fragments/form/featureChoiceFieldset.scala.html]: ^ Looks like the second outer if statement is now redundant, as it is implied in the following pattern matching...

**Comment by afiore** [comment]
:+1:

**Pull request merged**